source/extensions/filters/network/common/redis/fault_impl.h

```diff
@@ -0,0 +1,106 @@
+#pragma once
+
+#include <map>
+#include <string>
+#include <vector>
+
+#include "envoy/api/api.h"
+#include "envoy/extensions/filters/network/redis_proxy/v3/redis_proxy.pb.h"
+#include "envoy/upstream/upstream.h"
+
+#include "common/protobuf/utility.h"
+#include "common/singleton/const_singleton.h"
+
+#include "extensions/filters/network/common/redis/fault.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+namespace Common {
+namespace Redis {
+
+using FaultMap = std::unordered_map<std::string, std::vector<FaultSharedPtr>>;
```

Comment:
nit: use absl::flat_hash_map

Reply:
ping

Reply:
Done. Sorry github only shows some comments in the main view so I didn't see this one.